src/models/msg.js

```diff
@@ -46,6 +46,9 @@ class Msg {
 			this.type !== Msg.Type.ERROR &&
```

Comment:
It would probably be nice to create an array of types that we don't log and switch the code to

`return this.TYPES_NOT_TO_LOG.includes(this.type)` 

Might be nicer overall?